go.mod

```diff
@@ -3,19 +3,45 @@ module github.com/makerdao/vulcanizedb
 go 1.12
 
 require (
-	github.com/dave/jennifer v1.3.0
 	github.com/ethereum/go-ethereum v1.9.8
+	github.com/ClickHouse/clickhouse-go v1.4.3 // indirect
+	github.com/allegro/bigcache v1.2.1 // indirect
+	github.com/apilayer/freegeoip v3.5.0+incompatible // indirect
+	github.com/aristanetworks/goarista v0.0.0-20191106175434-873d404c7f40 // indirect
+	github.com/btcsuite/btcd v0.20.1-beta // indirect
+	github.com/cespare/cp v1.1.1 // indirect
+	github.com/dave/jennifer v1.3.0
+	github.com/deckarep/golang-set v1.7.1 // indirect
+	github.com/denisenkom/go-mssqldb v0.0.0-20200620013148-b91950f658ec // indirect
+	github.com/docker/docker v1.13.1 // indirect
+	github.com/edsrzf/mmap-go v1.0.0 // indirect
+	github.com/elastic/gosigar v0.10.5 // indirect
+	github.com/fjl/memsize v0.0.0-20190710130421-bcb5799ab5e5 // indirect
+	github.com/gballet/go-libpcsclite v0.0.0-20191108122812-4678299bea08 // indirect
+	github.com/gorilla/websocket v1.4.1 // indirect
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/hpcloud/tail v1.0.0
-	github.com/jmoiron/sqlx v0.0.0-20181024163419-82935fac6c1a
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/influxdata/influxdb v1.7.9 // indirect
+	github.com/jackpal/go-nat-pmp v1.0.2 // indirect
+	github.com/jmoiron/sqlx v1.2.0
+	github.com/karalabe/usb v0.0.0-20191104083709-911d15fe12a9 // indirect
 	github.com/lib/pq v1.0.0
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/onsi/ginkgo v1.10.1
 	github.com/onsi/gomega v1.10.0
-	github.com/pressly/goose v2.6.0+incompatible
+	github.com/oschwald/maxminddb-golang v1.5.0 // indirect
+	github.com/pborman/uuid v1.2.0 // indirect
+	github.com/pressly/goose v2.7.0-rc5+incompatible
+	github.com/prometheus/tsdb v0.10.0 // indirect
+	github.com/rjeczalik/notify v0.9.2 // indirect
+	github.com/rs/cors v1.7.0 // indirect
 	github.com/sirupsen/logrus v1.2.0
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/viper v1.3.2
+	github.com/status-im/keycard-go v0.0.0-20191119114148-6dd40a46baa0 // indirect
+	github.com/tyler-smith/go-bip39 v1.0.2 // indirect
+	github.com/ziutek/mymysql v1.5.4 // indirect
```

Comment:
assuming all these indirect deps came from goose - might be cool to see if a `go mod tidy` cleans things up

Reply:
👍